packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.module.css

```diff
@@ -52,7 +52,6 @@
   text-decoration-color: var(--primary-light-2);
 }
 .badge {
-  width: 12px;
```

Comment:
nit: I know it's annoying to swap branches etc, but would have done this in a separate branch so that this fix doesn't get blocked on other things